README.md

```diff
@@ -4,31 +4,42 @@ Resources used in the building of ecredis:
 - https://github.com/Tiroshan/eredis_cluster_client
 - https://github.com/Nordix/eredis_cluster
 
+ecredis is a high performance Erlang client for Redis Cluster. It uses eredis as the underlying
+implementation. Allows connecting to multiple redis clusters.
+
+## Features
+
+* Allows connections to multiple redis clusters at the same time
+* High throughput. The requests are executed directly on the callers process.
+
```

Comment:
should we mention that it does not work with a single client.

Reply:
Do you mean that it doesn't work for non clustered redis?

Reply:
I think it actually should because we have this function: get_cluster_slots_from_single_node(Node);